docs/application/native/guides/device/battery-monitor.md

```diff
@@ -39,37 +41,33 @@ To enable your application to use the Battery Monitor functionality:
 
 To get the battery usage information related to the application ID:
 
-- Get battery usage information of an application for a particular resource over a specific duration of time.
```

Comment:
AS IS:
https://docs.tizen.org/application/native/api/wearable/latest/group__CAPI__SYSTEM__BATTERY__BATTERY__MONITOR__MODULE.html
TO BE:
https://docs.tizen.org/application/native/api/wearable/5.5/group__CAPI__SYSTEM__BATTERY__BATTERY__MONITOR__MODULE.html

Both seem to be working. However I will modify it to "TO BE" as Battery Monitor is associated with Tizen-5.5.

Reply:
Please revert back to "latest", as the "latest" is symbolic link for 5.5.

The "latest" goes to the latest published API Reference version, and for now Tizen 5.5 is the latest published version. So we can consider "latest" as "5.5" until Tizen 6.0 is published.

In the  "view file" of the github repository, the symbolic link doesn't work. 
So in the "view file" the links seems broken, thus you should change the "latest" to actual latest version and check the link with it.

As @d1-agarwal said, https://docs.tizen.org/application/native/api/wearable/latest/group__CAPI__SYSTEM__BATTERY__BATTERY__MONITOR__MODULE.html is valid link.

@surbhig12 I think checking api links in the "view file" with changing "latest" to latest version is not spread to SRI-B reviewers well. Would you please share it once again?

Reply:
@mijin85cho  Please let me know how to check the working status of a symlink. 

Also, as @d1-agarwal  mentioned, it has "TO BE", that means it will work even if a new version is released, we do not have to change the file again from latest to 5.5 if we keep latest. 

@d1-agarwal correct me if  I am wrong in my understanding. 

So, @d1-agarwal if you agree with @mijin85cho opinion, then I request you to revert.

Reply:
@nawab-samsung 

> Please let me know how to check the working status of a symlink.

As I commented, you can check the working of the link changing the "latest" to "5.5" manually on "view file".
![image](https://user-images.githubusercontent.com/9958374/79434704-b7818080-8009-11ea-97df-42fd469764b8.png)

Or, you can use the staging link generated automatically, as @d1-agarwal did.

> Also, as @d1-agarwal mentioned, it has "TO BE", that means it will work even if a new version is released, we do not have to change the file again from latest to 5.5 if we keep latest.

As you said, one day a new version will be released, and at that time this link will guide the reader to "5.5". I think you can imagine the future that Tizen goes to 6.0. 6.5. 7.0.... and this link send reader to 5.5.
The reason we use symbolic link is to consider the new version update. To catch up the latest API for each guide pages, instead of hard cording the Tizen version, we use symbolic link. 
If you want to use "5.5", you have to check the links every time new Tizen version is released, and update each link manually. I think you can understand it is a load that could not be happen if we have used symbolic instead of hard cording.

Reply:
@mijin85cho got your point, staging was giving 404 that time. 
But other way of checking will be checking on live website, thats how @d1-agarwal checked. 
Thanks Both of you. 
@d1-agarwal  Kindly replace 5.5 with latest.

Reply:
Both the link seem to working for me. However I was not aware of the guidelines, thus I modified it considering the review comment. I will revert back. 
Thank you @mijin85cho and @nawab-samsung